src/Identity/ApiAuthorization.IdentityServer/src/Options/ApiScopeCollection.cs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using IdentityServer4.Models;
+
+namespace Microsoft.AspNetCore.ApiAuthorization.IdentityServer
+{
+    public class ApiScopeCollection : Collection<ApiScope>
```

Comment:
This just follows the pattern we follow across the other collection types in the options type.